source/features/tag-changes-link.tsx

```diff
@@ -33,8 +33,15 @@ async function getNextPage(): Promise<DocumentFragment> {
 }
 
 function parseTags(element: HTMLElement): TagDetails {
-	// Safari doesn't correctly parse links if they're loaded via AJAX #3899
```

Comment:
This is an old bug I can't repro anymore, so the comment and workaround can be dropped